playbook/app/pb_kits/playbook/utilities/accessibility.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-console */
+
+import axe from 'axe-core'
+
+export default function runAxe(include = '.pb--kit-example', ignore = ['.pb--kit-example :first-child']) {
```

Comment:
Is the space between example and : intended? `['.pb--kit-example :first-child']`

Reply:
Yep. This is the only way I could get Axe to ignore those captions inside each example.